tools/java-source-utils/java-source-utils.csproj

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
```

Comment:
`java-interop` is able to build for just "`net472`" and it works.  If possible it would be great to only build this once as well, as building java is much slower than .NET.

Reply:
Done in latest force-push.